analyzer/src/main/kotlin/managers/Pipenv.kt

```diff
@@ -72,7 +73,11 @@ class Pipenv(
 
         logger.info { "Generating '${requirementsFile.name}' file in '$workingDir' directory..." }
 
-        val requirements = ProcessCapture(workingDir, command(), "lock", "--requirements").requireSuccess().stdout
+        val requirements = if (Semver(getVersion()) >= Semver("2022.8.13")) {
```

Comment:
Should we maybe guard against `getVersion()` not being parsable as a `Semver` here? and maybe extract `Semver("2022.8.13")` to a class variable with a speaking name?

Reply:
Did you intend to also address the point from my first sentence?

Reply:
I forgot to respond, no, all versions of Pipenv so far are semver compliant so I don't think this is necessary.